tests/unittests/Foundation/NSSetTests.mm

```diff
@@ -148,4 +148,98 @@
     NSMutableSet* set = [NSMutableSet set];
     EXPECT_ANY_THROW([set addObject:nil]);
     EXPECT_EQ(0, [set count]);
-}
\ No newline at end of file
+}
+
+// Cannot be certain of order of elements so cannot compare directly with reference platform
```

Comment:
Isn't this comment no longer the case, now that the test no longer depends on ordering?